### PR TITLE
feat(dev/update) fixed description bug and redirected logo to home

### PR DESCRIPTION
**Pull request by Pendyala-thanvi**
### Functionality:
fixed description bug and redirected logo to home

### Solution:
fixed description bug and redirected logo to home

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/header/header.component.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class HeaderComponent implements OnInit {
 
   myDashBoard(){
     console.log("load MyDashboard");
-    this.router.navigate(["/MyDashboard"])
+    this.router.navigate(["/"])
```
Comment (k0malSharma) [inline src/app/header/header.component.ts:35]: Don't change the route here. This will create problem for going to MyDashboard component

**Review by k0malSharma — CHANGES_REQUESTED** [review]
Makes the required changes a

**Review by k0malSharma — APPROVED** [review]
Ship it

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**